openslides_backend/migrations/migrations/0034_rename_change_default_projector_ids.py

```diff
@@ -0,0 +1,56 @@
+from typing import Any, Dict, List, Optional
+
+from datastore.migrations import (
+    BaseEvent,
+    BaseMigration,
+    CreateEvent,
+    DeleteFieldsEvent,
+    ListUpdateEvent,
+    UpdateEvent,
+)
+from datastore.shared.util import collection_from_fqid
+
+
+class Migration(BaseMigration):
+    """
+    This migration renames default_projector_$.._id into
+      default_projector_$.._ids and changes value to list.
+    """
+
+    target_migration_index = 35
+    collection = "meeting"
+
+    def modify(self, object: Dict[str, Any]) -> None:
+        for field in list(object.keys()):
+            if field == "default_projector_$_id":
+                object[field + "s"] = object[field]
+                del object["default_projector_$_id"]
+            elif field.startswith("default_projector_$") and field.endswith("_id"):
+                old_field = field
+                new_field = field + "s"
+                if old_field in object:
+                    object[new_field] = [object[old_field]]
+                    del object[old_field]
```

Comment:
```suggestion
                object[field + "s"] = [object[field]]
                del object[field]
```
I would do it analogously to the first `if` and use the values directly. `old_field` is redundant anyway and doing the same thing twice in different ways is confusing.

Reply:
Also, the last `if` is redundant

Reply:
Done.